backend/de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/payment/paymentallocation/service/PaymentAllocationBuilder.java

```diff
@@ -404,6 +422,10 @@ else if (PayableRemainingOpenAmtPolicy.WRITE_OFF.equals(payableRemainingOpenAmtP
 	private List<AllocationLineCandidate> createAllocationLineCandidates_CreditMemosToInvoices(
 			@NonNull final List<PayableDocument> payableDocuments)
 	{
+		if(!allowInvoiceToCreditMemoAllocation)
```

Comment:
why are we checking allowInvoiceToCreditMemoAllocation here,
but we check allowPurchaseSalesInvoiceCompensation on upper level.

should we check all these flags on the same level?

Reply:
that's right! fixed.